sycl/test-e2e/KernelCompiler/kernel_compiler_basic.cpp

```diff
@@ -0,0 +1,79 @@
+//==- kernel_compiler_sycl_jit.cpp --- kernel_compiler extension tests -----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: (opencl || level_zero)
+// REQUIRES: aspect-usm_device_allocations
+
+// UNSUPPORTED: accelerator
+// UNSUPPORTED-INTENDED: while accelerator is AoT only, this cannot run there.
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 1
+// RUN: %{l0_leak_check} %{run} %t.out 1
```

Comment:
I realize that this is following a pattern that I set up, but these tests don't need both "run" and "run-with-leak-check".  Just running with leak check is enough.  On OpenCL (or any other non-supporting backend) the leak check env var will be ignored.

Let's drop the "run" lines.

And, while we are at it, these tests are also not using the arg, so we can drop the "1" at the end as well.